tigerpath/majors_and_certificates/scripts/verifier.py

```diff
@@ -392,7 +396,8 @@ def _init_path_to(req):
             _init_path_to(subreq)
 
 def _json_format(obj):
-   return json.dumps(obj, sort_keys=False, indent=2, separators=(',', ': ')) + "\n"
+    import json
```

Comment:
Is there a reason you moved the import here? Might be better to have it at the top of the file

Reply:
This is just a debugging function and is not called under normal usage, and this is the only place where json is needed.